### DISCUSSION TITLE
Only permit Tor

### DISCUSSION THREAD

**Discussion by aruizc13** [globaleaks/globaleaks-whistleblowing-software#4140] [Q&A]
Hi,

We are configuring Globalleaks and we would know if it´s possible to permit access only via Tor and not via normal navigation. 
Is there some option to do this configuration?

Thanks for all.

**Comment by evilaliv3** [comment]
Hello @aruizc13 

At the moment is not possible to limit the platform to be entierly used via Tor, but in the network settings you can prevent users to user the platform via normal network.

For example you can prevent whistleblowers to blow the whistle without using Tor.

**Comment by aruizc13** [comment]
Hi,

Ok, thanks for your time  and for all.
